tests/phpunit/elementor/core/breakpoints/test-breakpoint.php

```diff
@@ -0,0 +1,149 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\Core\Breakpoints;
+
+use Elementor\Core\Breakpoints\Breakpoint;
+use Elementor\Core\Breakpoints\Manager as Breakpoints_Manager;
+use Elementor\Plugin;
+use Elementor\Testing\Elementor_Test_Base;
+use Elementor\Testing\Traits\Extra_Assertions;
+
+class Test_Breakpoint extends Elementor_Test_Base {
+
+	use Extra_Assertions;
+
+	/**
+	 * Test Get Name
+	 *
+	 * Tests Breakpoint::get_name() for all breakpoints.
+	 *
+	 * @since 3.2.0
+	 */
+	public function test_get_name() {
+		$breakpoints = Plugin::$instance->breakpoints->get_breakpoints();
+
+		foreach ( $breakpoints as $breakpoint_name => $breakpoint ) {
+			/** @var Breakpoint $breakpoint */
+			$this->assertEquals( $breakpoint_name, $breakpoint->get_name() );
+		}
+	}
+
+	/**
+	 * Test Get Config
+	 *
+	 * Tests Breakpoint::get_config() for all breakpoints.
+	 *
+	 * @since 3.2.0
+	 */
+	public function test_get_config() {
+		$breakpoints = Plugin::$instance->breakpoints->get_breakpoints();
+
+		foreach ( $breakpoints as $breakpoint ) {
+			/** @var Breakpoint $breakpoint */
+			$breakpoint_config = $breakpoint->get_config();
+
+			$this->assertArrayHaveKeys( [ 'value', 'direction', 'is_enabled' ], $breakpoint_config );
+			$this->assertTrue( is_numeric( $breakpoint_config['value'] ) );
+			$this->assertTrue( is_string( $breakpoint_config['direction'] ) );
+			$this->assertTrue( is_bool( $breakpoint_config['is_enabled'] ) );
+		}
+	}
+
+	/**
+	 * Test Is Enabled
+	 *
+	 * Tests Breakpoint::get_config() for all breakpoints.
+	 *
+	 * @since 3.2.0
+	 */
+	public function test_is_enabled() {
```

Comment:
You should check this class as an agnostic component. Therefore, check if it's enabled by creating a new Breakpoint with `'is_enabled' => true`. The following test should be served as a test for `get_active_breakpoints()`

Reply:
The first part - done.
Using it with `get_active_breakpoints()` is currently not possible since we do not have an interface for adding new breakpoints externally. Only breakpoints that appear in the default config can be added to the Breakpoints Manager's breakpoint array.